done.js

```diff
@@ -3,241 +3,336 @@ var chroma = require('@v3rse/chroma');
 var moment = require('moment');
 //Path to task json file
 var TASK_JSON_PATH = "./.database.json";
-
+const exec = require('child_process').exec;
 
 
 //Creates a file for keeping track of tasks
 function init() {
-	//create file if it's present.
-	if (!fs.existsSync(TASK_JSON_PATH)) {
-		console.log("Initialising storage.\n Creating `.database.json` file");
-		setData({
-			uncompleted: [],
-			completed: []
-		});
-	}
+    //create file if it's present.
+    if (!fs.existsSync(TASK_JSON_PATH)) {
+        console.log("Initialising storage.\n Creating `.database.json` file");
+        setData({
+            uncompleted: [],
+            completed: []
+        });
+    }
 
 }
 
 
-
 //Used to read some data from the JSON file
 function getData() {
-	//read file contents
-	var contents = fs.readFileSync(TASK_JSON_PATH);
+    //read file contents
+    var contents = fs.readFileSync(TASK_JSON_PATH);
 
-	//parse contents
-	var data = JSON.parse(contents);
+    //parse contents
+    var data = JSON.parse(contents);
 
-	return data;
+    return data;
 }
 
 
-
 //Used to write data to the JSON file
 function setData(data) {
-	// makes the object a JSON string
-	var dataString = JSON.stringify(data);
+    // makes the object a JSON string
+    var dataString = JSON.stringify(data);
 
-	//write to  file
-	fs.writeFileSync(TASK_JSON_PATH, dataString);
+    //write to  file
+    fs.writeFileSync(TASK_JSON_PATH, dataString);
 }
 
 
-
 //Displays usage
 function usage() {
-	console.log("Usage: done [add|check|delete|help|clear [all|done]|list [all|done]] [task]");
-	console.log("`task` is only a string when using `add` and a number\nfor all other commands.");
-	console.log("Using the `done` without arguments lists all tasks");
+    console.log("Usage: done [add|check|delete|help|clear [all|done]|list [all|done]] [task]");
```

Comment:
Forgot to add this as well. We will need to add the `git` commands to the usage function